extensions/znotch/src/manage.ts

```diff
@@ -0,0 +1,19 @@
+import open from "open";
+import { closeMainWindow, showHUD } from "@raycast/api";
+import { isZNotchInstalled } from "./check-installed";
+
+export enum Action {
+  Toggle = "toggle",
+  Show = "show",
+  Hide = "hide",
+}
+
+export async function manage(action: Action, message: string) {
+  if (await isZNotchInstalled()) {
+    open(`xyz.kondor.znotch://v1/manage?action=${action}`);
```

Comment:
have you tried using the `open` system utility from Raycast? 

https://developers.raycast.com/api-reference/utilities#open